.gitignore

```diff
@@ -17,7 +17,6 @@
 .env.development.local
 .env.test.local
 .env.production.local
-.vscode
```

Comment:
We still want to ignore .vscode, I think this will persist though since it's in the main branch so no need to fix this here!

Reply:
Heard! I'm not sure why that was changed from my end. I don't think I messed with that on purpose